data_visualization_in_seaborn/data_visualization_in_seaborn.md

```diff
@@ -119,48 +120,57 @@ Alternatively, if you have the [Anaconda distribution of Python](https://www.ana
 </div>
 
 ### The data
-@sage
 
-First, we need to load the python modules we'll be using:
+First, we need to load the rest of the python modules we'll be using (we already loaded `seaborn` on the previous page, but it's run to run that code again):
```

Comment:
I think this is what you meant :) 
```suggestion
First, we need to load the rest of the python modules we'll be using (we already loaded `seaborn` on the previous page, but it's fun to run that code again):
```

Reply:
lol good catch, thank you! I meant "fine" actually :) Fixed now.